polkadot/xcm/pallet-xcm/src/lib.rs

```diff
@@ -2432,6 +2438,83 @@ impl<T: Config> Pallet<T> {
 		AccountIdConversion::<T::AccountId>::into_account_truncating(&ID)
 	}
 
+	/// Dry-runs `call` with the given `origin`.
```

Comment:
```suggestion
	/// Dry-runs `call` with the given `origin`.
	///
```

You should always have an empty line between the explanation and further docs. I don't have a link at hand that describes this. However, this way only the first line is rendered in a "coarse" overview and you see the rest when clicking on the function. Makes the docs looks better.